docs/breez-sdk/snippets/react-native/refunding_payments.ts

```diff
@@ -104,6 +105,25 @@ const setMaxFeeToRecommendedFees = () => {
   console.log('Config:', config)
 }
 
+const customClaimLogic = async (sdk: BreezSdk, deposit: DepositInfo) => {
+  // ANCHOR: custom-claim-logic
+  if (deposit.claimError instanceof DepositClaimError.MaxDepositClaimFeeExceeded) {
```

Comment:
I'm wondering if the `DepositClaimError_Tags` need to be used here. Not sure

```suggestion
  if (deposit.claimError.tag === DepositClaimError_Tags.MaxDepositClaimFeeExceeded) {
```